modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -347,6 +347,7 @@ public int compare(CodegenModel cm1, CodegenModel cm2) {
                 });
             }
         }
+        setCircularReferences(allModels);
```

Comment:
Can we add this to the default codegen instead as users would expect `isCircularReference` with a valid value (true, false) for all generators?

Reply:
@wing328 that was my intention, that's why I added it to `updateAllModels` in `DefaultCodegen`. Somehow this did got not triggered in Elm, so I thought I'd probably overridden it. Is `updateAllModels` the correct place to put this?

Reply:
Can you please move it to updateAllModels and I'll run some tests later to troubleshoot?

Reply:
@wing328 I already put it in `updateAllModels`. Do I need to remove it from the Elm codegen?